jenkins/pipelines/cd/tiup/tiup-mirror-update-multi-products.groovy

```diff
@@ -625,14 +599,11 @@ node("build_go1130") {
                             update_ctl RELEASE_TAG, "darwin", "amd64"
                             update "tidb", RELEASE_TAG, tidb_sha1, "darwin", "amd64"
                         }
-                    }
                 }
             }
             if (RELEASE_TAG >= "v5.1.0" || RELEASE_TAG == "nightly") { 
-                multi_os_update["TiUP build tidb on darwin/arm64"] = {
+                stage("TiUP build tidb on darwin/arm64") {
```

Comment:
I think put the if into the stage will be better.

Reply:
done